src/clr-addons/icons/cds-icons.ts

```diff
@@ -0,0 +1,6 @@
+import { ClarityIcons } from '@cds/core/icon';
+import { timelineIcon } from '@cds/core/icon/shapes/timeline';
+
+export function initCdsIcons(): void {
+  ClarityIcons.addIcons(timelineIcon);
+}
```

Comment:
We don't want to add icons into every bundle which uses clarity-addons. So if you need cds icons pls import them in your specific project (or better in the specific module inside the project, to keep bundle sizes small).